### DISCUSSION TITLE
read only

### DISCUSSION THREAD

**Discussion by jas0420** [dahlb/ha_carrier#48] [General]
Thank you for the Carrier integration!  I have had Home Assistant on mothballs for about a year now.  Had tried getting Infinitude going back then with mixed success.  I tried again today and I'm not getting anything.  I can tell that the thermostats are passing through the Infinitude proxy because I can still see them "online" in the Carrier app on my phone, but for whatever reason, Infinitude isn't catching the traffic.  Had noticed one brief mention that people were having problems after a certain firmware update, and I've been keeping mine updated, so probably that..............

ANYWAY....  I stumbled upon your API solution and it connected up right away and I can see all of the stats.  Cool!  

I was wondering if you know if those #$%& at Carrier changed up the API to be read only now???  I don't see any controls inside HA (temp or fan adjustment for example) and looking now at Carrier's own website (www.myinfinitytouch.carrier.com) I seem to only be able to read there as well...  Could have sworn I was able to control settings in there in the past.  The view below is all that I get, so I assume (if what I'm seeing is accurate) that the same limitations would apply to your work.  Such a frustrating company........................... !!!

(very happy to at least see stats on the units inside HA now thanks to your handywork, so thank you!)

![image](https://github.com/dahlb/ha_carrier/assets/98757424/dade6fc3-ccd0-476e-b725-3b4f4144ac8a)

**Comment by dahlb** [comment]
You should be able to update within ha, that website is always read only I think, but you can use the same API through the mobile app like https://play.google.com/store/apps/details?id=com.myinfinity which is able to update also

**Reply by jas0420** [reply]
I'm stupid.  Had just found the controls in HA as you posted.  Thank you again!

**Reply by dahlb** [reply]
![Screenshot_20230825-173932.png](https://github.com/dahlb/ha_carrier/assets/54117/d198d8ca-6e21-44e1-9704-38c04f4ea1f3)

This is what the update interface looks like in home assistant using this integration from my phone.

**Comment by jas0420** [comment]
Crap!!  I didn't scroll down far enough in Home Assistant, haha!  I only saw the stats section!  Found the control section and it works like a champ.....  you are awesome!!!!!!!!!!!!!!

![image](https://github.com/dahlb/ha_carrier/assets/98757424/6adfe007-9a4f-403a-8713-62fd6a65a47c)
